grafana/dashboards/subnets.json

```diff
@@ -1427,17 +1130,44 @@
       },
       "targets": [
         {
-          "exemplar": true,
-          "expr": "increase(avalanche_${subnet}_vm_rpcchainvm_bytes_to_id_cache_hit[5m])/(increase(avalanche_${subnet}_vm_rpcchainvm_bytes_to_id_cache_hit[5m])+increase(avalanche_${subnet}_vm_rpcchainvm_bytes_to_id_cache_miss[5m]))",
+          "datasource": {
+            "type": "prometheus"
+          },
+          "expr": "sum(increase(avalanche_${vm_type}_rpcchainvm_bytes_to_id_cache_get_count{chain=\"${chain_id}\", result=\"hit\"}[5m]) or 0 * up)",
```

Comment:
I think that once https://github.com/ava-labs/avalanchego/pull/3170 is merged it would make sense to make an rpcchainvm widget or dashboard.

The intent of the avalanchego metrics refactor was that `avalanche_VMNAME_...` would define metrics specific to that VM, and that there shouldn't be significant amounts of common components across different VMNAMEs. (which is why the rpcchainvm metrics change makes a lot of sense).

For example, I think we should make a subnet-evm dashboard and an xsvm dashboard. I don't think those dashboards should be related at all.